docs/api-reference/server/xviz-server.md

```diff
@@ -0,0 +1,57 @@
+# XVIZServer
+
+The XVIZServer will listen on a port for connections. Upon a connection the server will attempt to
+find a handler that can satisfy the request. If a handler is found it will be called to manage the
+request else the socket will be closed.
+
+## Example
+
+```js
+import {XVIZServer} from '@xviz/server';
+
+// Construction of 'handler'
+
+const ws = new XVIZServer([handler], options, () => {
+  console.log(`Listening on port ${ws.server.address().port}`);
+});
+```
+
+## Constructor
+
+## XVIZServer(handlers, options, cb)
+
+The `options` argument is passed through to the underlying Websocket server.
+
+Parameters:
+
+- `handlers` (Array) - Set of [XVIZHandler](/docs/api-reference/server/overview-handler.md)
+  instances to service requests
+- `options` (Object) - Options for the Server
+
+  - `port` (Number) - Port to listen on
+  - `maxPayload` (Number) - Port to listen on
```

Comment:
type here, this a websocket library param right?

Reply:
Correct.  Should specify details and size unit